app/Filament/Widgets/ProspectGrowthChart.php

```diff
@@ -67,7 +67,7 @@ protected function getOptions(): array
 
     protected function getData(): array
     {
-        $runningTotalPerMonth = Cache::tags(['prospects'])
+        $runningTotalPerMonth = Cache::tags(['{{prospects}}'])
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.